packages/strapi-plugin-upload/admin/src/containers/ModalStepper/index.js

```diff
@@ -255,6 +262,9 @@ const ModalStepper = ({
   };
 
   const handleSetCropResult = blob => {
+    // Emit event : the user cropped a file that is not uploaded
+    emitEvent('didCropFile', { duplicatedFile: null, location: 'upload' });
```

Comment:
I post the comment here, but the `did` event should be after the dispatch to ensure the job has been done. I might be wrong just asking :)

Reply:
Well we cannot have an error here as we are just setting a new file in our reducer.